doc/folders.md

```diff
@@ -386,3 +391,75 @@ for (BoxItem.Info itemInfo : itemsInFolder) {
     Metadata itemMetadata = itemInfo.getMetadata("properties", "global");
 }
 ```
+
+Create Cascade Policy On Folder
+-------------------------------
+
+To set a metadata policy, which applies metadata values on a folder to new items in the folder, call 
+[`create(BoxAPIConnection api, String folderID, String scope, String templateKey)`][create-policy]
+with the api connection, scope, template key of the metadata template to be cascaded, and the ID of the folder to apply the policy to.
+
+```java
+BoxMetadataCascadePolicy.Info metadataCascadePolicyInfo = BoxMetadataCascadePolicy.create(api, 'folder-id', 'metadata-scope', 'example-template-key');
+```
+
+[create-policy]: 
+
+Get a Cascade Policies Information
+----------------------------------
+
+To retrieve information about a specific metadata cascade policy, call 
+[`getInfo()`][get-info]
+
+```java
+BoxMetadataCascadePolicy metadataCascadePolicy = new BoxMetadataCascadePolicy(api, 'cascade-policy-id');
+BoxMetadataCascadePolicy.Info metadataCascadePolicyInfo = metadataCascadePolicy.getInfo();
+```
+
+[get-info]:
+
+Get All Cascade Policies on Folder
+----------------------------------
+
+To get a list of all cascade policies on a folder, which show the metadata templates that are being applied to all 
+items in the folder, call [`getAll(BoxAPIConnection api, String folderID)`][get-all] with the ID of the folder.
+
+```java
+BoxMetadataCascadePolicy.getAll(api, 'folder-id');
+```
+
+You can also set the `owner_enterprise_id` option to retrieve only cascade policies owned by a specific enterprise(defaults to the current enterprise).
+
+```java
+BoxMetadataCascadePolicy.getAll(api, 'folder-id', 'owner-enterprise-id', 100);
```

Comment:
What's the last parameter here?  The full signature of this method should be documented here somewhere.